src/widgets/arc_label/lv_arc_label.c

```diff
@@ -0,0 +1,637 @@
+/**
+ * @file lv_arc_label.c
+ *
+ */
+
+/*********************
+ *      INCLUDES
+ *********************/
+
+#include "lv_arc_label_private.h"
+
+#if LV_USE_ARC_LABEL != 0
+
+#include "../../core/lv_obj_class_private.h"
+#include "../../core/lv_obj_event_private.h"
+#include "../../core/lv_obj_private.h"
+#include "../../misc/lv_area_private.h"
+#include "../../misc/lv_assert.h"
+#include "../../misc/lv_text_private.h"
+
+#if LV_USE_FLOAT
+    #include <math.h>
+    #include <float.h>
+    #ifndef  M_PI
+        #define M_PI 3.14159265358979323846264338327950288
+    #endif
+#else
+    /* Use fixed point math for integer only platforms $ M_PI << 8 $ */
+    #define M_PI 804
+#endif
+
+/*********************
+ *      DEFINES
+ *********************/
+
+#define MY_CLASS (&lv_arc_label_class)
+
+#define LV_ARC_LABEL_DEBUG 0
```

Comment:
It would not be good to expose this define into the lv_conf / kconfig? developers may want to take the usage of this debug utillity in development phases.

Reply:
I think this is not necessary for most developers; it's just used to assist in future repairs to the arc label component itself.